### ISSUE TITLE
how to send as pickle format?

### ISSUE THREAD

**Issue by mgravlin** [daniellawrence/graphitesend#25]
I am trying to figure out how to send as pickle, I don't see anything in the documentation, but seems like it is supported looking over the source code. I tried a couple ways:

g = graphitesend.init(init_type='pickle')
g.send_list([('metric', 45), ('metric2', 55)])

Result: sends to port 2003 (not 2004) as large packets containing multiple metrics. Suppose metrics are separated by newlines. I see the metrics in Graphite.

g = graphitesend.init(init_type='pickle', graphite_port=2004)
g.send_list([('metric', 45), ('metric2', 55)])

Result: sends to port 2004 but not as pickle format? I do not see the metrics in Graphite.

How do you specify you want to send metrics using the pickle format?

**Comment by aliahsan** [comment]
@mgravlin based on experience, I think you might have installed 0.3.4 using pip and in that version even if you do
```python
g = graphitesend.init(init_type='pickle', graphite_port=2004)
```
it still uses GraphiteClient instead of GraphitePickleClient. This has been fixed in 0.4.0 but its available on pip yet. So you can do a custom install and it should work.

@daniellawrence making 0.4.0 available at pip will help some.

**Comment by daniellawrence** [comment]
@aliahsan thanks for looking after this for me :+1: 

I have released 0.4.0 to pypi to stop this from happening again.
